### DISCUSSION TITLE
红米电视，实时显示播放内容

### DISCUSSION THREAD

**Discussion by fay000fay** [XiaoMi/ha_xiaomi_home#521] [Ideas]
希望加入可以显示目前正在观看的APP，并可以手动选择打开某个APP，以及目前是否处于播放还是暂停状态。这样就可以做一些自动化了，比如当电视唤醒以后，自动打开某个APP。

**Comment by SusanPhevos** [comment]
xiaomi_home 根据设备的 MIoT-Spec-V2 定义生成相应的 HA 实体，厂商未在 MIoT-Spec-V2 中定义的功能无法在 xiaomi_home 中实现。
例如，[xiaomi.tv.rmaxv1](https://home.miot-spec.com/spec/xiaomi.tv.rmaxv1)，没有 playing-state 属性，因此 xiaomi_home 无法显示设备的播放状态。
